src/overlays/actors/ovl_Fuusen/ac_fuusen.c

```diff
@@ -22,34 +36,437 @@ ActorProfile Fuusen_Profile = {
     /* */ aFSN_actor_draw,
     /* */ NULL,
 };
-#endif
 
-#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Fuusen/ac_fuusen/aFSN_actor_ct.s")
+xyz_t Init_Size = { 0.01f, 0.01f, 0.01f };
```

Comment:
These are in-function static

Reply:
would this be the case for all data? not sure what the general ruling is for this on the repo

Reply:
If it has $ qualifier in the name from the maps, it's in function static, and generally we add them as in function static

Reply:
ah great, will adjust accordingly then!